content/scrum-master-training/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Scrum Master Training
+content_type: project
+submission_type: link
+---
+
+Complete the [Scrum Master Training](https://www.udemy.com/course/scrum-master-training/) once you have completed it upload a screenshot showing that it is complete to your Google Drive and submit.
```

Comment:
Some sentences can be a bit better. Eg "Complete the scrum master training" could rather be "complete the scrum master training course".